src/matter/cluster/AccessControlCluster.ts

```diff
@@ -0,0 +1,177 @@
+    /** Specifies a list of TargetStruct, which define the clusters on this Node to which this Access Control Entry grants access. */
+    targets: Field(4, ArrayT(TargetT)), /* nullable: true, maxArrayLength: targetsPerAccessControlEntry */
+});
+
+/** @see {@link MatterCoreSpecificationV1_0} § 9.10.5.4 */
+const AccessControlExtensionEntryT = ObjectT({
+    /** Used by manufacturers to store arbitrary TLV-encoded data related to a fabric’s Access Control Entries. */
+    data: Field(1, ByteStringT({ maxLength: 128 })),
+});
+
+
+/**
+ * The Access Control Cluster exposes a data model view of a Node’s Access Control List (ACL),
+ * which codifies the rules used to manage and enforce Access Control for the Node’s endpoints
+ * and their associated cluster instances. Access to this Access Control Cluster itself requires
+ * a special Administer privilege level, such that only Nodes granted such privilege (hereafter
+ * termed "Administrators") can manage the Access Control Cluster.
+ * The Access Control Cluster SHALL be present on the root node endpoint of each Node, and SHALL
+ * NOT be present on any other Endpoint of any Node.
+ *
+ * clusterRevision: 1
+ *
+ * @see {@link MatterCoreSpecificationV1_0} § 9.10
+ */
+export const AccessControlCluster = Cluster({
+    /** Is a base cluster, so no id */
+    id: 0x1f,
+    name: "Access Control",
+
+    /** @see {@link MatterCoreSpecificationV1_0} § 9.10.5 */
+    attributes: {
+        /** Codifies a single grant of privilege on this Node. */
+        acl: WritableAttribute(0, ArrayT(AccessControlEntryT), { default: [], writeAcl: AccessLevel.Administer, readAcl: AccessLevel.Administer }),
+        /** MAY be used by Administrators to store arbitrary data related to fabric’s Access Control Entries. */
+        extension: OptionalWritableAttribute(1, ArrayT(AccessControlExtensionEntryT), { default: [], writeAcl: AccessLevel.Administer, readAcl: AccessLevel.Administer }),
+        /** Provide the minimum number of Subjects per entry that are supported by this server. */
+        subjectsPerAccessControlEntry: Attribute(2, Bound(UInt16T,{ min: 4, max: 0xFFFF }), { default: 4 }),
+        /** Provides the minimum number of Targets per entry that are supported by this server. */
+        targetsPerAccessControlEntry: Attribute(3, Bound(UInt16T,{ min: 3, max: 0xFFFF }), { default: 3 }),
+        /** Provides the minimum number of ACL Entries per fabric that are supported by this server. */
+        accessControlEntriesPerFabric: Attribute(4, Bound(UInt16T,{ min: 3, max: 0xFFFF }), { default: 3 }),
+    },
+
+    /** @see {@link MatterCoreSpecificationV1_0} § 9.10.7 */
+    events: {
+        /**
+         * The cluster SHALL send AccessControlEntryChanged events whenever its ACL attribute data is changed by an
+         * Administrator.
+         * @see {@link MatterCoreSpecificationV1_0} § 9.10.7.1
+         */
+        accessControlEntryChanged: Event(0, EventPriority.Info, {
```

Comment:
To avoid repeating the common part with the next event, this can be generalized:
```
const AccessChangeEvent = <T>(entryTemplate: Template<T>) => ({
            /** The Node ID of the Administrator that made the change, if the change occurred via a CASE session. */
            adminNodeID: Field(0, NodeIdT), /* nullable: true */
            /** The Passcode ID of the Administrator that made the change, if the change occurred via a PASE session. */
            adminPasscodeID: Field(1, Bound(UInt16T,{ min: 0, max: 0xFFFF })), /* nullable: true */
            /** The type of change as appropriate. */
            changeType: Field(2, EnumT<ChangeTypeEnum>()),
            /** The latest value of the changed entry. */
            latestValue: Field(3, entryTemplate), /* nullable: true */
});
```